src/backend/d3d12/FramebufferD3D12.cpp

```diff
@@ -41,31 +40,26 @@ namespace d3d12 {
                 device->GetD3D12Device()->CreateRenderTargetView(texture.Get(), &rtvDesc, rtvHandle);
             } else {
                 // TODO(enga@google.com) no attachment. This will use the backbuffer. Remove when this hack is removed
-                usingBackbuffer = true;
                 rtvIndex++;
             }
         }
 
-        // TODO(enga@google.com): load depth attachment from subpass
-        if (usingBackbuffer) {
+        if (info.depthStencilAttachmentSet) {
```

Comment:
oops, you're right, this is totally wrong. Thanks for catching it. I'll do as you suggested.